projects/1.20-fabric/assets/cafes-birding/cafesbirding/lang/zh_cn.json

```
@@ -0,0 +1,146 @@
+  "item.minecraft.lingering_potion.effect.strong_satiation": "滞留型饱足药水",
+  "item.minecraft.tipped_arrow.effect.strong_satiation": "饱足之箭",
+  "effect.cafesbirding.umbra": "暗影",
+  "item.minecraft.potion.effect.umbra": "暗影药水",
+  "item.minecraft.splash_potion.effect.umbra": "喷溅型暗影药水 ",
+  "item.minecraft.lingering_potion.effect.umbra": "滞留型暗影药水",
+  "item.minecraft.tipped_arrow.effect.umbra": "暗影之箭",
+  "item.minecraft.potion.effect.long_umbra": "暗影药水",
+  "item.minecraft.splash_potion.effect.long_umbra": "喷溅型暗影药水 ",
+  "item.minecraft.lingering_potion.effect.long_umbra": "滞留型暗影药水",
+  "item.minecraft.tipped_arrow.effect.long_umbra": "暗影之箭",
+  "effect.cafesbirding.haleness": "强健",
+  "item.minecraft.potion.effect.haleness": "强健药水",
+  "item.minecraft.splash_potion.effect.haleness": "喷溅型强健药水 ",
+  "item.minecraft.lingering_potion.effect.haleness": "滞留型强健药水",
+  "item.minecraft.tipped_arrow.effect.haleness": "强健之箭",
+  "item.minecraft.potion.effect.long_haleness": "强健药水",
+  "item.minecraft.splash_potion.effect.long_haleness": "喷溅型强健药水 ",
+  "item.minecraft.lingering_potion.effect.long_haleness": "滞留型强健药水",
+  "item.minecraft.tipped_arrow.effect.long_haleness": "强健之箭",
+  "effect.cafesbirding.ravenous": "饥迫",
+  "item.minecraft.potion.effect.ravenous": "饥迫药水",
+  "item.minecraft.splash_potion.effect.ravenous": "喷溅型饥迫药水 ",
+  "item.minecraft.lingering_potion.effect.ravenous": "滞留型饥迫药水",
+  "item.minecraft.tipped_arrow.effect.ravenous": "饥迫之箭",
+  "item.minecraft.potion.effect.long_ravenous": "饥迫药水",
+  "item.minecraft.splash_potion.effect.long_ravenous": "喷溅型饥迫药水 ",
+  "item.minecraft.lingering_potion.effect.long_ravenous": "滞留型饥迫药水",
+  "item.minecraft.tipped_arrow.effect.long_ravenous": "饥迫之箭",
+  "effect.cafesbirding.arachnid": "蛛化",
+  "item.minecraft.potion.effect.arachnid": "蛛化药水",
+  "item.minecraft.splash_potion.effect.arachnid": "喷溅型蛛化药水 ",
+  "item.minecraft.lingering_potion.effect.arachnid": "滞留型蛛化药水",
+  "item.minecraft.tipped_arrow.effect.arachnid": "蛛化之箭",
+  "item.minecraft.potion.effect.long_arachnid": "蛛化药水",
+  "item.minecraft.splash_potion.effect.long_arachnid": "喷溅型蛛化药水 ",
+  "item.minecraft.lingering_potion.effect.long_rarachnid": "滞留型蛛化药水",
+  "item.minecraft.tipped_arrow.effect.long_arachnid": "蛛化之箭",
+  "item.cafesbirding.music_disc_feathers": "音乐唱片",
+  "item.cafesbirding.music_disc_feathers.desc": "Café - Feathers",
+  "item.cafesbirding.music_disc_grouch": "音乐唱片",
+  "item.cafesbirding.music_disc_grouch.desc": "Café - Grouch",
+  "item.cafesbirding.music_disc_pollen": "音乐唱片",
+  "item.cafesbirding.music_disc_pollen.desc": "Café - Pollen",
+  "item.cafesbirding.music_disc_nitid": "音乐唱片",
+  "item.cafesbirding.music_disc_nitid.desc": "Café - Nitid",
+  "item.cafesbirding.music_disc_ashes": "音乐唱片",
+  "item.cafesbirding.music_disc_ashes.desc": "Café - Ashes",
+  "item.cafesbirding.disc_fragment_nitid.desc": "音乐唱片 - Nitid",
+  "cafesbirding_sparrow_egg_item_lore_": "???",

```

```suggestion
  "cafesbirding_sparrow_egg_item_lore_": "？？？",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。

---

通查完毕，已修改